### PR TITLE
Add lecturas.md for giscedata_lectures_disri

### PR DESCRIPTION
**Pull request by MiquelIR**
References #169
Adds the remaining documentation of the giscedata lectures distri module

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_missing_docs_from_giscedata_lectures_distri/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_missing_docs_from_giscedata_lectures_distri/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_missing_docs_from_giscedata_lectures_distri/

File: docs/distri/lecturas.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+![](_static/lecturas/Captura06.png)
+
+
+És molt important que els impostos de client i proveïdor estiguin introduïts
+correctament. (A la imatge es mostra l'IVA vigent en aquell moment)
+
+!!! tip
+    Quan es creen productes, es recomana duplicar un producte de la mateixa
+    categoria que tingui tots els camps omplerts i així s'estalvia temps de
+    creació del producte. S'ha de fer des de `Formulari > Duplicar`.
```
Comment (jaumef) [inline docs/distri/lecturas.md]: No hauria de ser codi, en tot cas entre cometes
```markdown
`Formulari > Duplicar`
"Formulari → Duplicar"
```

File: docs/distri/lecturas.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+    Un cop duplicat s'han de canviar el _nom de producte_, el _codi_, el _preu_,
+    la _referència del proveïdor_ i el _proveïdor_ quan sigui necessari.
+
+La fitxa de producte es pot completar amb la pestanya `Preus & proveïdors` on
```
Comment (jaumef) [inline docs/distri/lecturas.md]: Fora el codi i el &
```markdown
`Preus & proveïdors`
_"Preus i proveïdors"_
```

File: docs/distri/lecturas.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+![](_static/lecturas/Captura07.png)
+
+
+* **Preu cost**: És el preu que s'utilitzarà a l'ordre de compra d'aquest
+  producte
+* **Mètode de cost**: Si en aquest camp es posa _Preu mig_ (serà el més
+  habitual) el preu de compra s'anirà actualitzant a partir dels diferents
+  preus de compra que es vagin donant.
+
+Les dades de les pestanyes `Proveïments i Ubicacions`, `Descripcions` i
```
Comment (jaumef) [inline docs/distri/lecturas.md]: El mateix amb el codi

File: docs/distri/lecturas.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+Es descriu com realitzar l'ordre de compra dels comptadors i la recepció del
+material en el magatzem.
+
+**_Mòdul de compres_**
+
+![](_static/lecturas/Captura01.png)
+
+
+Per començar es selecciona el menú `Compres > Comandes de compra > Sol·licitud
```
Comment (jaumef) [inline docs/distri/lecturas.md]: El codi no mola si no es codi

File: docs/distri/lecturas.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+dades es poden omplir les línies de la sol·licitud de pressupost seleccionant,
+de la llista de productes, el producte i la quantitat a sol·licitar.
+
+
+**_Afegir productes a sol·licitud de pressupost_**
+
+![](_static/lecturas/Captura05.png)
+
+
+Es selecciona el botó nou ![](_static/lecturas/iconaNou.png) de l'apartat `línia de ordre de compra`.
```
Comment (jaumef) [inline docs/distri/lecturas.md]: Mira al syllabus

File: docs/distri/lecturas.md [hunk 6]
```diff
@@ -0,0 +1,373 @@
...
+que ha d'omplir l'usuari.
+
+
+**_Omplir línia de compra_**
+
+![](_static/lecturas/Captura08.png)
+
+
+* **Producte**: Amb la lupa es selecciona el producte desitjat. Poden
+  utilitzar-se els filtres necessaris per filtrar (`categoria`,
```
Comment (jaumef) [inline docs/distri/lecturas.md]: Cap al codeblocks

File: docs/distri/lecturas.md [hunk 7]
```diff
@@ -0,0 +1,373 @@
...
+
+Un cop introduïdes totes les línies de sol·licitud de compra i confirmats els
+preus de compra es pot prèmer el el botó **Confirmar comanda de compra**
+
+!!! note
+    Es pot anul·lar posteriorment si es decideix no realitzar-la prement el botó
+    **Cancel·la comanda de compra**
+
+Un cop confirmada, es pot veure com l'estat de la comanda ha passat de l'estat
+`Petició de pressupost` a `confirmat` tal com es veu a la casella `Estat
```
Comment (jaumef) [inline docs/distri/lecturas.md]: amb un .sh et faran `rm -rf /*`

File: docs/distri/lecturas.md [hunk 8]
```diff
@@ -0,0 +1,373 @@
...
+de la comanda`
+
+
+**_Ordre de compra confirmada_**
+
+![](_static/lecturas/Captura10.png)
+
+
+Podem accedir posteriorment a les comandes confirmades a la opció de menú
+`Compres > Comandes de compra > Comanda de compra esperant aprovació` on es
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

File: docs/distri/lecturas.md [hunk 9]
```diff
@@ -0,0 +1,373 @@
...
+Fent doble clic sobre l'ordre de compra, es mostrarà en format formulari.
+
+Un cop fetes les gestions amb el proveïdor i es consensuin els preus amb la
+companyia, ja es pot prèmer el botó **Aprovada pel proveïdor**
+
+Mentre no es premi el botó **Aprovada pel Proveïdor** s'és a temps de modificar
+el preu dels productes o les seves quantitats.
+
+Un cop s'ha apretat el botó, l'ordre de compra passa a l'estat de `ordres de
+compra en procés`. Totes aquestes comandes les veurem al llistat accessible
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

File: docs/distri/lecturas.md [hunk 10]
```diff
@@ -0,0 +1,373 @@
...
+
+
+Aquesta fitxa conté un seguit de camps a títol informatiu:
+
+* **Ref. origen**: És el codi d'albarà en el qual ha arribat el material, per
+  defecte GISCE-ERP li assigna una referència de la comanda.
+* **Data prevista**: Data en la qual es fa la recepció del material. No es pot
+  modificar.
+* **Tipus d'enviament**: Quan el material arriba del proveïdor, sempre és
+  `Recepció mercaderies`
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

File: docs/distri/lecturas.md [hunk 11]
```diff
@@ -0,0 +1,373 @@
...
+
+
+**_Fitxa recepció material dividida en 1 sèrie correlativa (Cas 1)_**
+
+![](_static/lecturas/Captura19.png)
+
+
+Per acabar, prement el botó **Productes rebuts** apareixerà el cuadre de diàleg
+i prement el botó **Crear albarà** es donarà el procés de recepció de material
+per acabat, passant a estat `realitzat`.
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

File: docs/distri/lecturas.md [hunk 12]
```diff
@@ -0,0 +1,373 @@
...
+Suposem que arriben 4 comptadors amb número de sèrie correlatiu 10000100 al
+10000104, i un altre grup de 6 comptadors amb els números de sèrie de 20000200
+al 20000206.
+
+Es procedirà de la següent forma:
+
+Des de la fitxa de recepció de material haurem de dividir la línia de 10
+comptadors en dues línies, una de 6 comptadors i una altra de 4 comptadors.
+Seleccionem la línia i premem el botó d'**acció** ![](_static/lecturas/iconaAccio.png) i del menú que
+apareix seleccionem la opció `Divideix línia de moviment`.
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

File: docs/distri/lecturas.md [hunk 13]
```diff
@@ -0,0 +1,373 @@
...
+El resultat serà que es crearan 2 línies a la fitxa empaquetat/albarà.
+Apareixeràn les dues línies al fer doble-click sobre la línia.
+
+
+**_Recepció de material partida en dues línies (Cas 2)_**
+
+![](_static/lecturas/Captura24.png)
+
+
+Un cop divit l'`Empaquetat/Albarà` es procedirà com en el **Cas 1** per cada
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_missing_docs_from_giscedata_lectures_distri/

File: docs/distri/lecturas.md [hunk 14]
```diff
@@ -0,0 +1,373 @@
...
+correctament. (A la imatge es mostra l'IVA vigent en aquell moment)
+
+!!! tip
+    Quan es creen productes, es recomana duplicar un producte de la mateixa
+    categoria que tingui tots els camps omplerts i així s'estalvia temps de
+    creació del producte. S'ha de fer des de _"Formulari > Duplicar"_.
+    Un cop duplicat s'han de canviar el _nom de producte_, el _codi_, el _preu_,
+    la _referència del proveïdor_ i el _proveïdor_ quan sigui necessari.
+
+La fitxa de producte es pot completar amb la pestanya `Preus & proveïdors` on
```
Comment (jaumef) [inline docs/distri/lecturas.md]: :disappointed:

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_missing_docs_from_giscedata_lectures_distri/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_missing_docs_from_giscedata_lectures_distri/

**Pull request merged**